Components/Terrain/src/OgreTerrainLodManager.cpp

```diff
@@ -66,7 +66,14 @@ namespace Ogre
 
     bool TerrainLodManager::isOpen() const
     {
+#ifndef OGRE_HAS_CXX11_EXPLICIT_CONVERSION_OPERATORS
+#   pragma warning ( push )
+#   pragma warning ( disable: 4800 )
+#endif
         return (bool)mDataStream;
```

Comment:
I think `mDataStream != 0` would be a better fix instead of using the pragmas and associated compiler checks for this.

Reply:
I agree. I'll try this.